src/routes/devices/getAll.ts

```diff
@@ -32,17 +28,16 @@ export async function getAll (req, res): Promise<void> {
 
     for (const m of metadata) {
       list.push({
-        amtuser: amtCredentials ? amtCredentials[m.guid]?.amtuser : null,
-        conn: req.mpsService.mpsComputerList[m.guid] == null ? 0 : 1,
-        host: m.guid,
-        mpsuser: amtCredentials ? amtCredentials[m.guid]?.mpsuser : null,
-        name: amtCredentials ? amtCredentials[m.guid]?.name : null,
+        connectionStatus: req.mpsService.mpsComputerList[m.guid] == null ? 0 : 1,
```

Comment:
nit: can use ??

Reply:
what do you mean?

Reply:
actually it the `??` operator won't work here, my mistake